src/main/webapp/index.jsp

```diff
@@ -24,14 +33,14 @@
   <nav>
     <a id="navTitle" href="/">CodeU Chat App</a>
     <a href="/conversations">Conversations</a>
-    <% if(request.getSession().getAttribute("user") != null){ %>
-      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <a href="/about.jsp">About</a>
+    <a href="/testdata">Load Test Data</a>
+    <% if(loggedInUser != null){ %>
+      <a href= <%= "/profile/" + loggedInUser.getId().toString() %> > Hello <%= loggedInUser.getName() %>!</a>
```

Comment:
Oh that was a just a placeholder link so the user can see their profile page easier. Do you have any suggestions something that might sound better lol

Reply:
Ahh gotcha! That makes sense to me now. I thought that that "Hello name!" thing was rendered somewhere on the body of the webpage rather than in the navbar.